server/src/bootstrap/index.ts

```diff
@@ -77,7 +77,7 @@ app.use(requestLoggingMiddleware)
 
 // cors enable
 app.options('*', cors<express.Request>())
-app.use(cors({ origin: 'http://localhost:5000' }))
+app.use(cors({ origin: 'http://localhost:6174' }))
```

Comment:
Have this replaced by `SERVER_PORT` instead

Reply:
OK done!